engine/apps/grafana_plugin/helpers/client.py

```diff
@@ -337,6 +337,9 @@ def create_service_account_token(
     def get_service_account_token_permissions(self) -> APIClientResponse[typing.Dict[str, typing.List[str]]]:
         return self.api_get("api/access-control/user/permissions")
 
+    def setup_organization(self) -> APIClientResponse:
+        return self.api_post(f"api/plugins/{PluginID.ONCALL}/resources/plugin/sync?wait=true")
```

Comment:
We might also want `force=true` here as well

Reply:
Yeah, I wasn't sure about `force=true`, but maybe we can set it for now